java.base/src/main/resources/META-INF/qbicc/qbicc-patch-info

```diff
@@ -2,4 +2,6 @@
 
 jdk.internal.loader.BootLoader$_patch
 java.lang.Runtime$_runtime
+java.lang.Thread$_patch
 jdk.internal.misc.Unsafe$_runtime
+jdk.internal.misc.UnsafeParkUnpark
```

Comment:
Also add `jdk.internal.misc.UnsafeThreadAccess` to patch-info?

Reply:
Fixed.